### PR TITLE
feat: Add tags checking rule - allows specify rules for any tag and validate that

### PR DESCRIPTION
**Pull request by SqrTT**

### CODE REVIEW

**Comment by SqrTT** [comment]
Is this repo alive?

I would be appreciated for any feedback, thanks.

**Comment by SqrTT** [comment]
Anybody alive?

**Comment by dimanech** [comment]
It would be great if we have this functionality.

**Comment by SqrTT** [comment]
Unfortunately, looks like this repo is dead, hence, we have to move forward and switch to alive linter.

File: src/rules/tags-check.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    },
+    img: {
+        selfclosing: true,
+        attrsRequired: [
+            'src', 'alt', 'title'
+        ]
+    }
+};
+
+var assign = function assign(target) {
```
Comment (Shinigami92) [inline src/rules/tags-check.js]: I think that should be one of the following two options:
```js
function assign(target) {
```
or
```js
var assign = function(target) {
```

**Comment by SqrTT** [comment]
ping

**Comment by Gcaufy** [comment]
any process here?

**Comment by brettz9** [comment]
If this could be updated against `master` and accepted, it would seem based on the pull description that it would be a great last hurrah toward making the project customizable per user needs.

**Comment by thedaviddias** [comment]
Closing that issue, new and more recent PR opened #384

**Pull request closed without merging**